aks/README.md

```diff
@@ -0,0 +1,107 @@
+
+## Prerequisites
+1. Create a resource group if you haven't got one ready for use. 
+```bash
+az deployment create \
+  --name MY_RESOURCE_GROUP_NAME \
+  --location eastus \
+  --template-file ./resource-group.json
+```
+
+az deployment create \
+  --name joshua-rg8 \
+  --location eastus \
+  --template-file ./resource-group.json
+
+2. Don't create more than one AKS cluster in the same subnet.
+3. AKS clusters may **not** use _169.254.0.0/16, 172.30.0.0/16, 172.31.0.0/16, or 192.0.2.0/24_ for the Kubernetes service address range.
+4. To interact with Azure APIs, an AKS cluster requires an Azure Active Directory (AD) [Service Principal](https://docs.microsoft.com/en-us/azure/aks/kubernetes-service-principal). The service principal is needed to dynamically create and manage other Azure resources such as an Azure load balancer, container registry (ACR) etc
+
+Unless specified, the SP is assigned the 'Contributor' role by default - this has full permissions to read and write to your Azure account which isn't ideal from a security standpoint.
```

Comment:
Please define SP as you do ACR above before using it for the first time

Reply:
done